diff -Naur old/CMakeLists.txt new/CMakeLists.txt
--- old/CMakeLists.txt	2021-04-26 18:03:21.371691107 +0800
+++ new/CMakeLists.txt	2021-04-26 18:05:13.247323434 +0800
@@ -204,8 +204,8 @@
 
 if(ENABLE_OPENGL)
   if(ENABLE_OPENGLES2)
-    pkg_check_modules(GLESV2 REQUIRED glesv2)
-    set(HAVE_OPENGL TRUE)
+	  find_package(PkgConfig REQUIRED)
+	  set(HAVE_OPENGL TRUE)
     set(OPENGL_INCLUDE_DIR  ${GLESV2_INCLUDE_DIRS})
     set(OPENGL_LIBRARY ${GLESV2_LIBRARIES})
     add_definitions(-DUSE_OPENGLES2)
