diff -Naur old/es-core/src/platform.cpp new/es-core/src/platform.cpp
--- old/es-core/src/platform.cpp	2022-05-06 16:08:38.000000000 +0800
+++ new/es-core/src/platform.cpp	2022-05-10 14:54:46.877435045 +0800
@@ -32,7 +32,7 @@
     system("/usr/bin/emuelec-utils small-cores enable");
     return system("systemctl poweroff");
 #else // osx / linux	
-	return system("shutdown -h now");
+	return system("/usr/bin/xbox360-controllers-shutdown ; shutdown -h now");
 #endif
 }
 
