diff -Naur old/es-app/src/guis/GuiMenu.cpp new/es-app/src/guis/GuiMenu.cpp
--- old/es-app/src/guis/GuiMenu.cpp	2022-10-18 12:01:07.000000000 +0800
+++ new/es-app/src/guis/GuiMenu.cpp	2022-10-18 12:06:35.642050723 +0800
@@ -854,7 +854,7 @@
 		else
 		{
 #ifdef _ENABLEEMUELEC	
-		mVersion.setText("EMUELEC ES V" + ApiSystem::getInstance()->getVersion() + buildDate + " IP:" + getShOutput(R"(/usr/bin/emuelec-utils getip)"));
+		mVersion.setText("Taiwan. EMUELEC H3 V" + ApiSystem::getInstance()->getVersion() + buildDate + " IP:" + getShOutput(R"(/usr/bin/emuelec-utils getip)"));
 #else
 			std::string aboutInfo = ApiSystem::getInstance()->getApplicationName() + " V" + ApiSystem::getInstance()->getVersion();
 			mVersion.setText(aboutInfo + buildDate);
